tests/CacheCompat/CommonCache.Test.MsalJava/pom.xml

```diff
@@ -24,12 +22,12 @@
         <dependency>
             <groupId>com.microsoft.azure</groupId>
             <artifactId>msal4j</artifactId>
-            <version>0.6.0-preview</version>
+            <version>1.6.1</version>
```

Comment:
>1.6.1 [](start = 21, length = 5)

@petro, @avery : please ensure this is updated as part of Java releases.

Reply:
If we want this to always be updated as part of a release, couldn't we just have this dependency in the .pom to be ```<version>[1.6)</version>```

Reply:
Nice idea about grabbing latest version. 
<version>LATEST</version> is deprecated in maven 3
Recommended approach is using  "Versions Maven Plugin" - http://www.mojohaus.org/versions-maven-plugin/ 
+ passing versions:update-properties argument

Reply:
Thanks @Avery-Dunn , I think your suggestion is the simplest and will go with it!